docs/src/content/docs/reference/overrides.md

```diff
@@ -0,0 +1,374 @@
+---
+title: Overrides Reference
+description: Learn
```

Comment:
Was preparing a draft french translation of the docs/guides ahead of time while commuting and noticed this description.

I guess this one is incomplete and should be something like "An overview of the components and components props supported by Starlight overrides."?

Reply:
Good catch! 😅 

I like your suggestion.
```suggestion
description: An overview of the components and component props supported by Starlight overrides.
```